cpp/ql/test/query-tests/Likely Bugs/Format/WrongTypeFormatArguments/Linux_mixed_byte_wprintf/WrongTypeFormatArguments.expected

```diff
@@ -1,5 +1,7 @@
 | tests.cpp:18:15:18:22 | Hello | This argument should be of type 'char *' but is of type 'char16_t *' |
 | tests.cpp:19:15:19:22 | Hello | This argument should be of type 'char *' but is of type 'wchar_t *' |
+| tests.cpp:21:15:21:21 | Hello | This argument should be of type 'char16_t *' but is of type 'char *' |
+| tests.cpp:21:15:21:21 | Hello | This argument should be of type 'wchar_t *' but is of type 'char *' |
```

Comment:
Do you know how this will render in LGTM? One message with two values, or two separate messages on the same line?

Reply:
On LGTM it'll be one alert with an expandable placeholder that says `[ 2 values ]`. [Here's](https://lgtm.com/projects/g/apache/hive/snapshot/fdcbe7485b1f317a42bd59c6c0212bd526edc565/files/standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java#x863fca6f0efdbba4:1) an example from a Java query.